ScoutSuite/providers/gcp/facade/iam.py

```diff
@@ -0,0 +1,30 @@
+from ScoutSuite.providers.gcp.facade.facade import Facade
+from ScoutSuite.providers.gcp.facade.utils import GCPFacadeUtils
+from ScoutSuite.providers.utils import run_concurrently
+
+class IAMFacade(Facade):
+    def __init__(self):
+        super(IAMFacade, self).__init__('iam', 'v1')
+
+    async def get_bindings(self, project_id, service_account_email):
+        resource = 'projects/{}/serviceAccounts/{}'.format(project_id, service_account_email)
+        iam_client = self._get_client()
+        response = await run_concurrently(
+                lambda: iam_client.projects().serviceAccounts().getIamPolicy(resource=resource).execute()
+        )
+        return response.get('bindings', [])
+
+    async def get_keys(self, project_id, service_account_email):
+        name = 'projects/{}/serviceAccounts/{}'.format(project_id, service_account_email)
+        iam_client = self._get_client()
+        response = await run_concurrently(
+                lambda: iam_client.projects().serviceAccounts().keys().list(name=name).execute()
+        )
+        return response.get('keys', [])
+
+    async def get_service_accounts(self, project_id):
+        name = 'projects/{}'.format(project_id)     
+        iam_client = self._get_client()
+        request = iam_client.projects().serviceAccounts().list(name=name)
```

Comment:
I think this should be parallelized!

```suggestion
        request = await run_concurrently(
                iam_client.projects().serviceAccounts().list(name=name)
        )
```

Reply:
`request` and `services_accounts_group` simply build the request from my understanding. The actual fetching is done in `GCPFacadeUtils.get_all()` which is already parallelized .